src/import-utils.ts

```diff
@@ -217,6 +236,35 @@ export function rewriteAstBaseUrl(
 }
 
 /**
+ * Walk through inline scripts of an import document.
+ * For each script create identity source maps unless one already exists.
+ *
+ * The generated script mapping detail is the relative location within
+ * the script tag. Later this will be updated to account for the
+ * line offset within the final bundle.
+ */
+export async function addSourcemapsToInlineScripts(
+    document: Document, ast: ASTNode, oldBaseUrl: UrlString) {
+  const inlineScripts = dom5.queryAll(ast, matchers.inlineJavascript);
+  const parsedHtmlDocument = document.parsedDocument as ParsedHtmlDocument;
+  const promises = inlineScripts.map(scriptAst => {
+    let content = dom5.getTextContent(scriptAst);
+    const sourceRange = parsedHtmlDocument.sourceRangeForStartTag(scriptAst)!;
```

Comment:
https://github.com/Polymer/polymer-bundler/pull/449/files#diff-66e48e66925dfeda3f9ec59c717ffd32R104 reparses it as a fragment